README.md

```diff
@@ -1172,6 +1172,7 @@ Update Time, five active automations, webhooks.
   * [DronaHQ](https://www.dronahq.com/) — DronaHQ - a low code platform that helps engineering teams and product managers to build internal tools, custom user journeys, digital experiences, automation, custom admin panels, operational apps 10X faster.
   * [ILLA Cloud](https://www.illacloud.com/) — ILLA Cloud - A robust open-source low-code platform for developers to build internal tools. By using ILLA's library of Components and Actions, developers can save massive amounts of time on building tools. Free for 5 team members.
   * [outsystems.com](https://www.outsystems.com/) — Enterprise web development PaaS for on-premise or cloud, free "personal environment" offering allows for unlimited code and up to 1 GB database
+  * [apibrew.io](https://apibrew.io/) - ApiBrew is a Low Code platform to build Rest APIs from either UI or from yaml files. You can also customize the behaviour of APIs with Javascript codes.
```

Comment:
thanks, can you briefly state whats free pls, thanks

Reply:
Add Shotcut Track- track.shotcut.in
Shotcut Track is a web analytics and user behavior that complies with GDPR. 
 This is Software as a Service not self hosted
 It has a free tier, not just a free trial
 Pricing information is clearly visible without signup or phone calls
 The submission is not already present in the list
 The service has contact details of those running it and a privacy policy
 Articles are available on How to use it.
thansk

Reply:
@tansihmittal additions are done using pull requests, thanks